src/app/jscaip/MoveCoordToCoord.ts

```diff
@@ -3,18 +3,9 @@ import { Direction } from './Direction';
 import { MGPFallible } from '../utils/MGPFallible';
 import { RulesFailure } from './RulesFailure';
 import { MoveWithTwoCoords } from './MoveWithTwoCoords';
-import { Encoder } from '../utils/Encoder';
 
-export class MoveCoordToCoord extends MoveWithTwoCoords {
+export abstract class MoveCoordToCoord extends MoveWithTwoCoords {
 
-    public static encoder: Encoder<MoveCoordToCoord> = Encoder.tuple(
-        [Coord.encoder, Coord.encoder],
-        (move: MoveCoordToCoord) => [move.getStart(), move.getEnd()],
-        (fields: [Coord, Coord]) => MoveCoordToCoord.of(fields[0], fields[1]));
-
-    public static of(start: Coord, end: Coord): MoveCoordToCoord {
-        return new MoveCoordToCoord(start, end);
-    }
     public constructor(start: Coord, end: Coord) {
```

Comment:
Tiens, je revient sur tout mes commentaires "override is bettar plz", MAIS
vu qu'on est plutôt en train d'aller dans la direction de la souplesse de config, MoveCoordToCoord gagne a rester non abstrait non ?

Reply:
je propose de le rendre non abstrait quand on en aura vraiment besoin ? Ici je pensais en avoir besoin mais au final non, je vois pas trop d'use case où on y gagne à l'avoir en concret en fait